msal/application.py

```diff
@@ -22,7 +22,7 @@
 
 
 # The __init__.py will import this. Not the other way around.
-__version__ = "1.31.0"  # When releasing, also check and bump our dependencies's versions if needed
+__version__ = "1.32.0"  # When releasing, also check and bump our dependencies's versions if needed
```

Comment:
**issue:** Please revert this change. We shall only bump version number during a release process. That way, a potential revert of a feature branch would not mess up the version number (because version numbers - once released - can never be reverted).

Reply:
done